src/store/plugins/sdk13.js

```diff
@@ -0,0 +1,205 @@
+import {
+  AeSdkWallet,
+  sign,
+  Node,
+  WALLET_TYPE,
+  messageToHash,
+  unpackTx,
+  RpcRejectedByUserError,
+  CompilerHttp,
+} from '@aeternity/aepp-sdk-13';
+import { isEmpty, isEqual } from 'lodash-es';
+import { App } from '../modules/permissions';
+import { showPopup } from '../../background/popupHandler';
+import { MODAL_CONFIRM_CONNECT, watchUntilTruthy } from '../../popup/utils';
+import { IS_CORDOVA, IS_EXTENSION_BACKGROUND } from '../../lib/environment';
+
+export default (store) => {
+  let sdk;
+  const aeppInfo = {};
+
+  store.registerModule('sdk13Plugin', {
+    namespaced: true,
+    state: {
+      ready: false,
+    },
+    getters: {
+      sdk: ({ ready }) => (ready ? sdk : undefined),
+    },
+    mutations: {
+      setSdkReady(state) {
+        state.ready = true;
+      },
+    },
+    actions: {
+      async initialize({ commit }) {
+        if (sdk) return;
+        await watchUntilTruthy(
+          () => store.state.isRestored && !isEmpty(store.getters.account),
+        );
+
+        const { activeNetwork } = store.getters;
+
+        // TODO: use this method for `onMessageSign` replacement.
+        // eslint-disable-next-line no-unused-vars
+        const signCb = async (payload, options, origin = null) => {
+          try {
+            const host = origin ? new URL(origin).host : null;
+            const permission = (!host && IS_CORDOVA)
+              ? true
+              : await store.dispatch('permissions/checkPermissions', {
+                host,
+                method: 'sign',
+                params: payload,
+              });
```

Comment:
I think you could simplify it to
```suggestion
            const permission = (!host && IS_CORDOVA) || await store.dispatch('permissions/checkPermissions', {
                host,
                method: 'sign',
                params: payload,
              });
```